### DISCUSSION TITLE
Can the size of training images and the size of images for inference differ?

### DISCUSSION THREAD

**Discussion by paul-reiners** [MIC-DKFZ/nnUNet#837] [Q&A]
Can the size of training images and the size of images for inference differ?

**Comment by paul-reiners** [comment]
When running nnUNet on a new dataset one of the things laid out is "Dataset Conversion" and a link to a document is provided (here). Line 121-122 of this document states " For each training case, all images must have the same geometry to ensure that their pixel arrays are aligned. Also make sure that all your data is co-registered! ".

**Reply by FabianIsensee** [reply]
No. Image sizes can be whatever.
What this statement says is that all images that belong to the same training case (for example multiple MRI modalities that are used together like in BraTS) must have the same geometry. Different training cases can have different geometries no problem

[a comment by an automated account is not shown]
